libdokan/tlf.go

```diff
@@ -165,11 +165,16 @@ func (tlf *TLF) open(ctx context.Context, oc *openContext, path []string) (dokan
 		tlf.refcount.Increase()
 		return tlf, true, nil
 	}
-	dir, exitEarly, err := tlf.loadDirAllowNonexistent(ctx, "open")
+	// If it is a creation then we need the dir for real.
+	dir, exitEarly, err := tlf.loadDirHelper(ctx, "open", !oc.isCreation())
 	if err != nil {
 		return nil, false, err
 	}
 	if exitEarly {
+		if node := openSpecialFile(lastStr(path), tlf.folder); node != nil {
```

Comment:
Not sure it does much good to allow the user to open a special file like `.kbfs_status` here, if there's no TLF to populate the data. What's the reasoning behind having this check?